src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -42,6 +42,15 @@ class RsHighlightingAnnotator : AnnotatorBase() {
         holder.newSilentAnnotation(severity).textAttributes(color.textAttributesKey).create()
     }
 
+    // For elements in a RsMacroExpansionReferenceGroup or RsMacroBindingGroup, return a macro color.
+    // Otherwise, return null
+    private fun macroGroupColor(parent: RsElement, element: PsiElement): RsColor? {
+        return if (parent is RsMacroExpansionReferenceGroup || parent is RsMacroBindingGroup)
+            RsColor.MACRO
+        else
+            null
+    }
```

Comment:
Let's keep `{}` or write it in one line

Reply:
Actually, I'd personally write it in the following way:

```kotlin
private val RsElement.isMacroGroup: Boolean get() = this is RsMacroExpansionReferenceGroup || this is RsMacroBindingGroup
```

and below
```kotlin
MUL, PLUS, LPAREN, LBRACE, RPAREN, RBRACE -> if (parent.isMacroGroup) RsColor.MACRO else null
```

Reply:
My plan was that is that in a future PR there will probably be different colors for expansions and patterns, respectively.